mbtb_app/web_portal/mbtb_portal/assets/js/suspend_user_form.js

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Where is button click event? this code should be executed once the admin press a button.